src/app/common/hooks/use-submit-stx-transaction.ts

```diff
@@ -53,7 +57,11 @@ export function useSubmitTransactionCallback({ loadingKey }: UseSubmitTransactio
               txId: safelyFormatHexTxid(response.txid),
             });
             await delay(500);
-            toast.success('Transaction submitted!');
+            const toastMessage = isIncreaseFeeTransaction
```

Comment:
We could define this as a function outside the hook?

Reply:
and pass in as callback 👍🏼

Reply:
I believe better to put in onSuccess callback, that was good idea